Workbooks/WorkbooksMetadata.json

```diff
@@ -2295,5 +2295,18 @@
       "templateRelativePath": "LogAnalyticsQueryAnalysis.json",
       "subtitle": "",
       "provider": "Microsoft Sentinel Community"
-    } 
+    },
+    {
+      "workbookKey": "AsdEssential8",
+      "logoFileName": "",
+      "description": "This workbook provides insights on the health state of Azure resources against requirements by the ASD Essential 8.",
+      "dataTypesDependencies": [ "DeviceTvmSecureConfigurationAssessment" ],
+      "dataConnectorsDependencies": [],
+      "previewImagesFileNames": [ "AsdEssential8Black1.png", "AsdEssential8White1.png", "AsdEssential8Black2.png", "AsdEssential8White2.png" ],
+      "version": "1.0.0",
+      "title": "ASD Essential 8",
+      "templateRelativePath": "AsdEssential8.json",
+      "subtitle": "",
+      "provider": "Microsoft Corporation"
```

Comment:
@nickt444, to keep it consistent, can you please just "Microsoft" rather than "Microsoft Corporation", thanks.

Reply:
done